### PR TITLE
Implement try catch strategy ManagementController

### PR DESCRIPTION
**Pull request by Machi3mfl**
Hi team,
This PR implement our new error handling strategy on ManagementController

Close: #3396

### CODE REVIEW

File: public/controllers/management/management.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export class ManagementController {
     this.logtestOpened = false;
     this.uploadOpened = false;
     this.rulesetTab = RulesetResources.RULES;
+    this.logger = new ErrorOrchestratorService();
```
Comment (frankeros) [inline public/controllers/management/management.js]: it should be implemented with getErrorOrchestratorService(), right?

File: public/controllers/management/management.js [hunk 2]
```diff
@@ -248,16 +277,25 @@ export class ManagementController {
...
+      const errorOptions = {
+        context: 'restartCluster',
+        location: 'ManagementController',
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        display: true,
+        store: true,
+        error: {
+          error: error,
+          message: error?.message,
```
Comment (frankeros) [inline public/controllers/management/management.js]: different implementation than line 164

File: public/controllers/management/management.js [hunk 3]
```diff
@@ -392,26 +429,36 @@ export class ManagementController {
...
+      const errorOptions = {
+        context: 'loadNodeList',
+        location: 'ManagementController',
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        display: true,
+        store: true,
+        error: {
+          error: error,
+          message: error?.message,
```
Comment (frankeros) [inline public/controllers/management/management.js]: different implementation than line 164

File: public/controllers/management/management.js [hunk 4]
```diff
@@ -392,26 +429,36 @@ export class ManagementController {
...
+          title: error?.name,
```
Comment (frankeros) [inline public/controllers/management/management.js:432]: different implementation than the others, the title could be undefined

File: public/controllers/management/management.js [hunk 5]
```diff
@@ -453,24 +500,39 @@ export class ManagementController {
...
       ErrorHandler.info('Upload successful');
       return;
     } catch (error) {
-      if (Array.isArray(error) && error.length) return Promise.reject(error);
-      ErrorHandler.handle('Files cannot be uploaded');
+      if (Array.isArray(error) && error.length) throw error;
+
+      const errorOptions = {
+        context: 'uploadFiles',
+        location: 'ManagementController',
```
Comment (frankeros) [inline public/controllers/management/management.js]: `'ManagementController'` is repeated in all catch, I suggest set it in a const

File: public/controllers/management/management.js [hunk 6]
```diff
@@ -207,26 +205,40 @@ export class ManagementController {
...
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
+          error: error,
+          message: error?.message || '',
+          title: 'Error restarting cluster',
+        },
+      };
+
+      ErrorOrchestratorService().handleError(errorOptions);
```
Comment (gabiwassan) [inline public/controllers/management/management.js:240]: Check the ultimates changes of `4.3-7.10` now you can use this service like this:
`getErrorOrchestrator().handleError(errorOptions);`

**Review by gabiwassan — CHANGES_REQUESTED** [review]

**Review by gabiwassan — APPROVED** [review]
LGTM! 👍🏼

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**